Lib/test/test_doctest.py

```diff
@@ -2231,11 +2231,9 @@ def test_DocFileSuite():
        files that include examples:
 
          >>> import unittest
-         >>> suite = doctest.DocFileSuite('test_doctest.txt',
-         ...                              'test_doctest2.txt',
-         ...                              'test_doctest4.txt')
+         >>> suite = doctest.DocFileSuite('doctest_DocFileSuite_test.txt',)
```

Comment:
Is there a reason this was changed?

Reply:
I think it should be a simple loader test. Multi file test has been done several times below.
If you think it's unreasonable, I can roll back~~